lib/crawling/suwon-dto.ts

```
@@ -0,0 +1,62 @@
+import type { Course, Credit, Student } from '@/types/domain';
+import type { CourseDTO, CreditDTO, StudentDTO } from '@/types/dto';
+
+/** StudentDTO → Student 변환 */
+function mapStudentDTOToDomain(dto: StudentDTO): Student {
+  return {
+    studentId: dto.sno, // 학번을 그대로 사용
+    portalUsername: dto.sno, // 학번을 포털 유저네임으로 사용
+    name: dto.studNm, // 학생 이름
+    departmentId: parseInt(dto.dpmjCd, 10), // 학과 ID
+    departmentName: dto.dpmjNm, // 학과 이름
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)